app/main/__init__.py

```diff
@@ -36,6 +36,7 @@
 
 content_loader.load_manifest('g-cloud-10', 'services', 'edit_service')
 content_loader.load_manifest('g-cloud-10', 'services', 'edit_submission')
+content_loader.load_manifest('g-cloud-10', 'services', 'copy_services')
```

Comment:
This will be `content_loader.load_metadata('g-cloud-10', ['copying-services'])` now.